### ISSUE TITLE
Open source aplication

### ISSUE THREAD

**Issue by danielmotaleite** [tonyp13/DragonDisk#21]
As useful app is not being maintained anymore, please consider open source it, so other can pick it up and fix the bugs

thanks!! :)

**Comment by benmctee** [comment]
Yes - please open source! If you aren't going to make improvements please let someone else take over so that we can have a decent client.
